android/quest/src/androidTest/java/org/smartregister/fhircore/quest/ui/shared/components/ViewGeneratorTest.kt

```diff
@@ -180,6 +180,30 @@ class ViewGeneratorTest {
       .assertIsDisplayed()
   }
 
+  @Test
+  fun testColumnIsRenderedCorrectlyWithDivider() {
+    composeRule.setContent {
+      GenerateView(
+        properties =
+          ColumnProperties(
+            wrapContent = false,
+            alignment = ViewAlignment.CENTER,
+            arrangement = ColumnArrangement.TOP,
+            showDivider = "true",
```

Comment:
Could you add a test case for the negative assertion here? That is a new test case with **divider** set to `false`